react/src/components/DisplayStatusText.js

```diff
@@ -0,0 +1,35 @@
+import Styles from '../Styles';
+import Str from '../utils/Str';
+
+const DisplayStatusText = ({ status }) => {
+    status = status?.toLowerCase(status).trim() || ""
+    function isStatusOkay(status) {
+        return status == "ok" || status == "done" || status.includes("pass") || status.includes("success");
+    }
+    function isStatusWarning(status) {
+        return status == "warn" || status == "warning"
+    }
+    function isStatusError(status) {
+        return status.includes("error") || status.includes("fail") || status.includes("exception");
+    }
+    function isStatusUnknown(status) {
+        return !Str.HasValue(status);
+    }
+    if (isStatusOkay(status)) {
+        return <b style={{color:Styles.GetForegroundColor()}}>OK</b>
```

Comment:
so the user doesn't need to know the specific value?

Reply:
I don't think so, IMO. There is the ability in the UI to see the raw JSON if they want to.